content/src/service/deployments/deployments.ts

```diff
@@ -0,0 +1,67 @@
+import { EntityId, Pointer } from 'dcl-catalyst-commons'
+import { AppComponents } from 'src/types'
+import { getPointerChangesForDeployments } from '../../logic/deployment-deltas-queries'
+import { getHistoricalDeployments } from '../../logic/deployments-queries'
+import { DELTA_POINTER_RESULT } from '../pointers/PointerManager'
+import {
+  DeploymentPointerChanges,
+  PartialDeploymentPointerChanges,
+  PointerChanges,
+  PointerChangesOptions
+} from './types'
+
+const MAX_HISTORY_LIMIT = 500
+
+export async function getPointerChanges(
```

Comment:
Is `deployments.ts` file the correct place for this function? Or maybe it should be in a separate file?